core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/util/base/RightV4.java

```diff
@@ -10,16 +10,22 @@
 import com.sequenceiq.cloudbreak.exception.NotFoundException;
 
 public enum RightV4 {
+    // account level
     ENV_CREATE(AuthorizationResourceAction.CREATE_ENVIRONMENT),
+    // env level
     ENV_START(AuthorizationResourceAction.START_ENVIRONMENT),
     ENV_STOP(AuthorizationResourceAction.STOP_ENVIRONMENT),
     ENV_DELETE(AuthorizationResourceAction.DELETE_ENVIRONMENT),
+    ENV_CHANGE_CRED(AuthorizationResourceAction.CHANGE_CREDENTIAL),
     DH_CREATE(AuthorizationResourceAction.CREATE_DATAHUB),
+    // dh level
     DH_START(AuthorizationResourceAction.START_DATAHUB),
     DH_STOP(AuthorizationResourceAction.STOP_DATAHUB),
     DH_DELETE(AuthorizationResourceAction.DELETE_DATAHUB),
+    // sdx level
     SDX_UPGRADE(AuthorizationResourceAction.UPGRADE_DATALAKE),
-    CHANGE_CRED(AuthorizationResourceAction.CHANGE_CREDENTIAL),
```

Comment:
please do not delete this because it will not be backward compatible, keep it and move it at the end of the list